src/Sidekick/Windows/Overlay/OverlayController.cs

```diff
@@ -1,70 +1,123 @@
 using System;
+using System.Diagnostics;
 using System.Linq;
+using System.Threading.Tasks;
 using System.Windows;
 using System.Windows.Forms;
 using System.Windows.Input;
 using Sidekick.Business.Apis.Poe.Models;
+using Sidekick.Business.Apis.PoeNinja;
+using Sidekick.Business.Apis.PoePriceInfo.Models;
+using Sidekick.Business.Parsers;
+using Sidekick.Business.Trades;
 using Sidekick.Business.Trades.Results;
+using Sidekick.Core.Loggers;
+using Sidekick.Core.Natives;
+using Sidekick.Core.Settings;
+using Sidekick.Localization;
+using Application = System.Windows.Application;
 using Cursor = System.Windows.Forms.Cursor;
 
 namespace Sidekick.Windows.Overlay
 {
-    public static class OverlayController
+    public class OverlayController
```

Comment:
We want to move away from using Controllers. We want to use ViewModels that are located in the UI project. You can take a look at the SettingsViewModel and View to see how to make it work.

Reply:
Domialex is okay with merging as it is. But more work will need to be done to move most of the code to view models. Let me know if you want to fix this now or if you want us to merge.

Reply:
@leMicin moving the code to view models is a different effort. This was just to remove the service locator stuff. 

There are probably a bunch of use cases that I did not test (because I don't know what they are) so somethings may be broken. If you are cool with that, I'm good. If/when those are found, happy to pitch in to fix them.